packages/config/config/devices/0x027a/zse50.json

```
@@ -0,0 +1,269 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZSE50",
+	"description": "Siren & Chime",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0369"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "On-Off Control",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Set Siren Playback Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 3,
+			"options": [
+				{
+					"label": "Play once",
+					"value": 0
+				},
+				{
+					"label": "Play in a loop for the duration per parameter two",
+					"value": 1
+				},
+				{
+					"label": "Play the loop per the number set in parameter three",
+					"value": 2
+				},
+				{
+					"label": "Play in a loop until cancelled by user",
+					"value": 3
+				},
+				{
+					"label": "No sound, LED indicator only",
+					"value": 4
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Set Playback Duration for the Siren",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 900,
+			"defaultValue": 180
+		},
+		{
+			"#": "3",
+			"label": "Set the Number of Playback Loops",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 1
+		},
+		{
+			"#": "4",
+			"label": "Default File Number Tone for the Siren Playback",
```

This sounds weird. Maybe 
```suggestion
			"label": "Default Siren Playback Tone File Number",
```